src/Reflection/BetterReflection/SourceLocator/ComposerJsonAndInstalledJsonSourceLocatorMaker.php

```diff
@@ -33,24 +36,21 @@ public function __construct(
 
 	public function create(string $projectInstallationPath): ?SourceLocator
 	{
-		$composerJsonPath = $projectInstallationPath . '/composer.json';
-		if (!is_file($composerJsonPath)) {
+		$composer = ComposerHelper::getComposerConfig($projectInstallationPath);
+
+		if ($composer === null) {
 			return null;
 		}
-		$installedJsonPath = $projectInstallationPath . '/vendor/composer/installed.json';
+
+		$this->vendorFolder = ComposerHelper::getVendorDir($projectInstallationPath, $composer);
```

Comment:
I don't want to nit-pick too much, but the correct term is `directory`. Here's an explanation: https://stackoverflow.com/questions/5078676/what-is-the-difference-between-a-directory-and-a-folder

In short - "directory" is the correct term, "folder" is what Windows calls that 😂

Reply:
And I think it would be nicer not to use a property for that, it looks too stateful to me. It's better to pass it around as a method parameter :) Thanks!

Reply:
Done